openml-lightgbm/pom.xml

```diff
@@ -1,213 +1,26 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why did you remove this?

Reply:
Thanks Sheng! I had to create a new pom.xml, and forgot to add the license and that. Fixed in the new commit!